mkdocs.yml

```diff
@@ -64,24 +64,30 @@ nav:
               - Base Definitions: reference/legacy/built-in/base.md
   - Metrics Glossary:
       - Metrics Glossary: metrics/index.md
-      - metrics/accuracy.md
-      - metrics/average-precision.md
-      - metrics/averaging-methods.md
-      - metrics/bertscore.md
-      - metrics/bleu.md
-      - metrics/confusion-matrix.md
-      - metrics/f1-score.md
-      - metrics/fpr.md
-      - metrics/geometry-matching.md
-      - metrics/iou.md
-      - metrics/meteor.md
-      - metrics/precision.md
-      - metrics/pr-curve.md
-      - metrics/recall.md
-      - metrics/roc-curve.md
-      - metrics/rouge-n.md
-      - metrics/specificity.md
-      - metrics/tp-fp-fn-tn.md
+      - Classification Metrics:
+        - Classification Metrics: metrics/classification/index.md
```

Comment:
Feels like we can omit these — for metrics that are neither CV nor NLP specific, leaving them under the toplevel "Metrics Glossary" header is fine.